Patternfly 6 upgrade

jasonmadigan opened the pull request:
- Upgrades libraries and CSS from PatternFly 5 to PatternFly 6
- Resolves some bugs with state and topology filtering

R-Lawton commented:
Few things i noticed 
* The http route and policies card on the overview used to be side by side now there one above the other.

pf6:
<img width="2222" height="794" alt="Screenshot 2025-07-17 at 12 00 27" src="https://github.com/user-attachments/assets/51ce584d-4bfb-486a-b430-317ee4309440" />

pf5:
<img width="2224" height="671" alt="Screenshot 2025-07-17 at 12 00 41" src="https://github.com/user-attachments/assets/f4a470e2-a62b-4b6a-9259-fd546b919b60" />


* The tab headings in the policies the font is quite small compared to before
pf6
<img width="977" height="312" alt="Screenshot 2025-07-17 at 11 57 49" src="https://github.com/user-attachments/assets/b32fa345-0a87-4b69-a52b-4b0aec608df5" />


* This is more of a nit pick but the values in all the tables seem to be slightly higher in the rows and not centerd compared to pf5

pf5:
<img width="1316" height="188" alt="Screenshot 2025-07-17 at 12 06 00" src="https://github.com/user-attachments/assets/485c25e2-9fbc-4c50-a0ea-f58bddb6eebb" />

pf6:
<img width="1273" height="207" alt="Screenshot 2025-07-17 at 12 05 48" src="https://github.com/user-attachments/assets/27e1bd86-b3eb-4033-a538-a4434d69aca6" />

jasonmadigan commented:
Thanks @R-Lawton 

> The http route and policies card on the overview used to be side by side now there one above the other.

Fixed

> The tab headings in the policies the font is quite small compared to before

Seems to be a general PF6 typography change, so I'll probably keep as-is for the sake of general console consistency. See `/settings/cluster`:

<img width="909" height="299" alt="Screenshot 2025-07-21 at 11 13 51" src="https://github.com/user-attachments/assets/4b45db3d-3f5b-4073-b2af-d350bd72732e" />

> This is more of a nit pick but the values in all the tables seem to be slightly higher in the rows and not centerd compared to pf5

looks odd to me too, addressed now

R-Lawton reviewed (approved):
/lgtm verified locally
